app/templates/Members/auto_complete.php

```diff
@@ -1,7 +1,7 @@
 <?php
 foreach ($query as $sca_member) :
     //split up the word by the search term so it can be highlighted
-    $sca_name = preg_replace('/(' . $q . '|' . $nq . '|' . $uq . ')/i', '<span class="text-primary">$1</span>', h($sca_member->sca_name)); ?>
+    $sca_name = preg_replace('/(' . $q . '|' . $nq . '|' . $uq . ')/i', '<span class="text-primary">$1</span>', $sca_member->sca_name); ?>
```

Comment:
Does parsing the html special chars here cause an issue w/ the autocomplete feature?

Reply:
No I've tested that.. .its a pain but it works...

Reply:
Its been a while since I messed with that but if I remember correctly its actually the single quote that gets messed up.